packages/plugin-commands-outdated/src/outdated.ts

```diff
@@ -244,6 +251,31 @@ ${renderCurrent(outdatedPkg)} ${chalk.grey('=>')} ${renderLatest(outdatedPkg)}`
     .join('\n\n') + '\n'
 }
 
+function renderOutdatedJSON (outdatedPackages: readonly OutdatedPackage[], opts: { long?: boolean }) {
+  const outdatedPackagesJSON = {} as {
+    [outdatedPackageName: string]: {
+      current: string
+      latest: string
+      dependencyType: DependenciesField
+      details?: string
+    }
+  }
+  for (const outdatedPkg of sortOutdatedPackages(outdatedPackages)) {
+    const { packageName, belongsTo } = outdatedPkg
+    outdatedPackagesJSON[packageName] = {
+      current: renderCurrent(outdatedPkg),
+      latest: extractLatest(outdatedPkg),
+      dependencyType: belongsTo,
+    }
+
+    if (opts.long) {
+      outdatedPackagesJSON[packageName].details = extractDetails(outdatedPkg)
```

Comment:
I think for long we can just pass the whole latest manifest
```suggestion
      outdatedPackagesJSON[packageName].latestManifest = outdatedPkg.latestManifest
```

Reply:
same as https://github.com/pnpm/pnpm/pull/5582#discussion_r1014654608